lib/jira/commands/transition.rb

```diff
@@ -33,7 +35,10 @@ def run
     private
 
       def params
-        { transition: { id: transition } }
+        {
+          transition: { id: transition[:id] },
+          fields: resolution_required?(transition) ? { resolution: { name: resolution } } : {}
```

Comment:
we're using `resolution_required?(transition)` twice per transition - we only really need to call it once. will post a simplification below